src/impls.rs

```diff
@@ -435,7 +433,9 @@ safety_comment! {
 
 safety_comment! {
     /// SAFETY:
-    /// TODO(#896): Write this safety proof before the next stable release.
+    /// Function objects (`fn()` and `extern "C" fn()`) self-evidently do not
```

Comment:
```suggestion
    /// Function pointers (`fn()` and `extern "C" fn()`) self-evidently do not
```
Function *objects* describes the broader set that includes closures, which may contain `UnsafeCell`s.